lte/gateway/fabfile.py

```diff
@@ -579,6 +579,7 @@ def get_test_summaries(
 
 def get_test_logs(
     gateway_host=None,
+    gateway_host_name='magma',
```

Comment:
Could we call this with `get_test_logs(gateway_host=magma_deb)` instead of introducing a new parameter?

Reply:
I don't see how this can be done without a (imho hard to read/understand) if-else part where `gateway_host_name` is currently used. I.e. something like
```
if not gateway_host:
        setup_env_vagrant("magma")
        gateway_host = env.hosts[0]
else if gateway_host == 'magma_deb':
        setup_env_vagrant("magma_deb")
        gateway_host = env.hosts[0]
```
The semantic of `gateway_host` would be:
* None -> defaults to magma -> get magma ssh host string
* magma_deb -> get magma_deb ssh host string
* foo@bar:123 -> use ssh host string directly

But I am open for a suggestion - maybe I am not seeing the elegant solution.

Reply:
Right, I didn't see that the ssh host string strictly requires the `foo@bar:123` format.